### ISSUE TITLE
Pop animations not working

### ISSUE THREAD

**Issue by mrousavy** [wix/react-native-navigation#6582]
### Issue Description

When animating a screen using custom push/pop animations, the push animations run perfectly fine. But once the screen gets dismissed, the pop animation will not get executed, only a simple alpha animation is being run:

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/15199031/93062599-7bfce800-f675-11ea-9677-6c36fef94c4f.gif)

### Steps to Reproduce / Code Snippets / Screenshots

1. Create Screen A and Screen B
2. Navigate from Screen A -> Screen B:

```ts
Navigation.push(componentId, {
	component: {
		name: 'ProfileScreen',
		options: {
			animations: {
				push: {
					waitForRender: true,
					content: {
						translationY: {
							from: height,
							to: 0,
							interpolation: 'decelerate',
						},
					},
				},
				pop: {
					waitForRender: true,
					content: {
						translationY: {
							from: 0,
							to: height,
							interpolation: 'decelerate',
						},
					},
				},
			},
		},
	},
});
```

3. Go back from Screen B -> Screen A:

```ts
Navigation.pop(componentId);
```

---
### Environment
* React Native Navigation version: 7.0.0
* React Native version: 0.63.2
* Platform(s) (iOS, Android, or both?): iOS, haven't tested on Android

**Comment by joeshub** [comment]
Correct me if I'm wrong, from what I can tell in only using this for a few days is that you can't animate pop, or at least you can't change its default behavior

**Comment by mrousavy** [comment]
Has been fixed in 7.1.0.
